src/sparql/aba_mapping_report.sparql

```diff
@@ -1,16 +1,26 @@
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX oio: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#> 
 
-SELECT DISTINCT ?supname ?sup ?subname ?sub
+
+SELECT DISTINCT ?supname ?sup ?subname ?sub ?inferred
 WHERE {
   { ?sub rdfs:subClassOf ?sup .
     ?sub rdfs:label ?subname .
     ?sup rdfs:label ?supname .
+      OPTIONAL {
+      [a owl:Axiom ;
+           owl:annotatedSource ?sub ;
+           owl:annotatedProperty rdfs:subClassOf ;
+           owl:annotatedTarget ?sup ;
+           oio:is_inferred ?inferred ] .
+           }
     }
    FILTER (STRSTARTS(STR(?sup), "http://purl.obolibrary.org/obo/UBERON_"))
    # Can't get double filter to work!
```

Comment:
@dosumis The "UBERON_0002616" filter technically can work, however, many classes are mapped to "UBERON_0002616" (wrongly so for now) - adding this filter would remove a lot of classes that are in and need fixing rather than removal.

Reply:
I thought the idea was to remove very general classifications that come in through inference, rather than assertion as with this.

Reply:
there is, strangely enough, 8266 terms that are subclass of http://purl.obolibrary.org/obo/UBERON_0002616, all of which are inferred. I've uploaded the terms here: https://github.com/obophenotype/ABA_Uberon/blob/b1a0daed3376f10719c6d8dc2d246772bb5bf157/src/ontology/report/uberon0002616.tsv